submissions/onyemeMichael/database/transportationQuery.sql

```diff
@@ -0,0 +1,40 @@
+/** Total number of passengers who suvrvived **/
+SELECT COUNT(survived) FROM accidents WHERE survived = 1;
+/** answer: 269 **/
```

Comment:
Your answer is incorrect

I guess you didn't import your data correctly. To even confirm, the total number of records on the CSV file for survival sheet is 891. If your answer is correct when you sum the two results you have, it ought to give you that total (i.e 891). But 269 + 419 = 688 instead

Reply:
The csv file for survival sheet which we download is 688.

![survival_import](https://user-images.githubusercontent.com/52843057/98959222-ef757700-24b7-11eb-96a2-899ba48e1f9c.png)

Reply:
check the sheet and see the total number of data there

Reply:
![image](https://user-images.githubusercontent.com/43061141/99080278-621e4980-25c1-11eb-9907-18b85908491d.png)

Reply:
Remember we are both importing the same record, I will adviced you DROP your table, create the table again and import the data gain.